packages/api/src/components/Migrator.ts

```diff
@@ -119,7 +119,10 @@ export class Migrator {
 					}
 				}
 			}
-			await this.db.collection(collectionName).insertMany(docs)
```

Comment:
the create_mock_data script should be run before the migrator pops off

Reply:
Yes, I was hesitant to include this but if the mock data is set to create 0 docs for any collection. (contacts and engagements in my case) it stops running the script

Reply:
Yep! I had my create_mock_data set up to not create contacts and engagements and having 0 docs in a collection stopped the entire script